src/integrations/eventorigin.ts

```diff
@@ -0,0 +1,28 @@
+import { EventProcessor, Integration } from '@sentry/types';
+
+/** Default EventOrigin instrumentation */
+export class EventOrigin implements Integration {
+  /**
+   * @inheritDoc
+   */
+  public static id: string = 'EventOrigin';
+
+  /**
+   * @inheritDoc
+   */
+  public name: string = EventOrigin.id;
+
+  /**
+   * @inheritDoc
+   */
+  public setupOnce(addGlobalEventProcessor: (e: EventProcessor) => void): void {
+    addGlobalEventProcessor(event => {
+      event.tags = event.tags ?? {};
+
+      event.tags['event.origin'] = 'javascript';
+      event.tags['event.environment'] = 'javascript';
```

Comment:
What other value can this be?
I'm not clear on the difference between these two tags

Reply:
@bruno-garcia `event.origin` currently would be either `javascript` or `android`. But for `event.environment` we have that to distinguish between ndk and java like below. `event.environment` was added later and is not present on the JS layer for RN and Cordova. I added it here for consistency so that the tag is present on all events.

https://github.com/getsentry/sentry-capacitor/blob/8e50fc8159ae757e98d54534ffc539194a6b7b9b/android/src/main/java/io/sentry/capacitor/SentryCapacitor.java#L320-L340

Reply:
That makes sense, thanks